reference/library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -152,10 +152,12 @@ body.#{$namespace}toolMenu-collapsed{
 		list-style: none;
 		padding: 0 0 0 0;
 	}
+	@media #{$phone}{
```

Comment:
I wondered if it's helpful to be able to quickly tell where the subsite buttons are and where the tool buttons are, rather than having to scan all the icons.